src/Middleware/src/Headstart.API/Controllers/BuyerLocationController.cs

```diff
@@ -51,7 +51,7 @@ public async Task CreatePermissionGroup(string buyerID, string buyerLocationID,
         [HttpPut, Route("{buyerID}/{buyerLocationID}"), OrderCloudUserAuth(ApiRole.UserGroupAdmin, ApiRole.AddressAdmin)]
         public async Task<HSBuyerLocation> Save(string buyerID, string buyerLocationID, [FromBody] HSBuyerLocation buyerLocation)
         {
-            return await _buyerLocationCommand.Save(buyerID, buyerLocationID, buyerLocation, UserContext.AccessToken);
+            return await _buyerLocationCommand.Save(buyerID, buyerLocationID, buyerLocation, UserContext.AccessToken, _oc);
```

Comment:
Looks like there are already two Save functions in the BuyerLocationCommand with different signitures. One doesn't include an ordercloud client as a parameter. Could we just be sure we are calling that function so we don't have to pass in  a client? 
![image](https://user-images.githubusercontent.com/31425559/117671265-38a83a00-b16e-11eb-9dc0-9d111b867944.png)